### PR TITLE
Add a button to request approveacls per branch

### PR DESCRIPTION
**Pull request by pypingou**
Fixes https://github.com/fedora-infra/pkgdb2/issues/210

### CODE REVIEW

File: pkgdb2/templates/package.html [hunk 1]
```diff
@@ -394,13 +394,17 @@ <h4 title="`approveacls` ACL">Package Administrator(s)</h4>
               </form>
             </li>
           {% endif %}
-          {% if g.fas_user.username in admins or is_admin %}
+          {% if g.fas_user %}
             <li>
               <a href="{{ url_for('.update_acl',package=package.name,
                        update_acl='approveacls') }}"
-                title="Manage commit ACLs">
```
Comment (ralphbean) [inline pkgdb2/templates/package.html]: So, this text was incorrect before, right?  The link url_for says 'approveacls', but this text says "commit".

File: pkgdb2/templates/package.html [hunk 2]
```diff
@@ -394,13 +394,17 @@ <h4 title="`approveacls` ACL">Package Administrator(s)</h4>
...
+                title="Manage approveacls ACLs">
                 <button class="bold pkg_button_blue inline wheel">
-                  Manage the admins
+                {% if g.fas_user.username in admins or is_admin%}
+                Manage the admins
+                {% else %}
+                Request Approveacls
```
Comment (ralphbean) [inline pkgdb2/templates/package.html]: Will this really just work if you are not an admin?  Hitting the same link will request the approveacls acls for you?

Comment (pypingou) [inline pkgdb2/templates/package.html]: It shows the page where you can request your ACLs

For admins it show all the users with that ACL on the package

So yes :)

File: pkgdb2/templates/package.html (hunk 1 above)
Comment (pypingou) [inline pkgdb2/templates/package.html]: Yes  it was incorrect

**Comment by ralphbean** [comment]
:+1:

**Pull request merged**